pytest.ini

```diff
@@ -10,3 +10,4 @@ markers =
     services: mark as a services test
     user: mark as a test for a user
     slow: mark a test as slow
+    server: mark as a test for a server
```

Comment:
Note that I'm not sure how useful this is because while it allows running just the server annotated tests, because of how the tests are setup to not be idempotent just running `pytest -m server` will fail, so this isn't very useful right now but seemed like the pattern to copy.